tests/Silex/Tests/Provider/ValidatorServiceProviderTest.php

```diff
@@ -55,6 +56,21 @@ public function testRegisterWithCustomValidators()
         return $app;
     }
 
+    public function testRegisterWithAnnotationLoader()
```

Comment:
About adding this as a new option instead of a service: Its possible to override it already, the documentation doesn't state it.
I have added an option only for commodity and because I think more users would benefit from an option that is so common, instead of overwriting the whole service.